### DISCUSSION TITLE
Modifying Wiki through PR's?

### DISCUSSION THREAD

**Discussion by iamthesenate1** [lxqt/lxqt#2013] [Q&A] [answered]
I was wandaring if the wiki can be modiffied through PR's.

**Comment by stefonarch** [comment]
It should work, just fork it to your git (up on the right is the  button).

**Reply by iamthesenate1** [reply]
When I click it, it just takes me to my fork, without the wiki.

**Reply by stefonarch** [reply]
Looks like you're right, I tried and the fork is just about lxqt and the wiki says " Welcome to the lxqt wiki!" and is empty.
Some people have write access, but I've actually no idea who can manage those permissions.

**Reply by iamthesenate1** [reply]
> Some people have write access, but I've actually no idea who can manage those permissions.

Apparantly I have write acces. I asked because, in my opinion changes should be approved first by members of LXQt.

**Reply by tsujan** [reply]
That may be a problem with GitHub's Wikis. If everyone has the write access, there might be a mess after a while; if we limit the write access to the members, people will have no way of modifying Wikis (am I right?).

**Reply by tsujan** [reply]
> I asked because, in my opinion changes should be approved first by members of LXQt.

You're completely right. Thanks for the consideration!

**Reply by stefonarch** [reply]
As there are no PR or drafts but only an "revert" possibility maybe a dedicated discussion here would be helpful. 
Everytime someone changes something 
* make a note at the bottom about the changes
* ping it in the dedicated discussion

**Comment by stefonarch** [comment, marked as answer]
End of story, no, in can't be done by PR's.

**Comment by Chiitoo** [comment]
Not directly, no, but with a bit more work, I believe it should be possible (to some extent at least) if one were to clone the wiki as a git repository, and make changes to that, then push it out for review.  Perhaps there could be a separate repository just for this, which would make PRs possible.  I think.

If I remember right, I've only cloned and pushed to a GitHub wiki to rewrite the history, so I've not actually done anything exactly like this myself.

I did find an answer at stackoverflow which seems to be talking about something similar to what I'm thinking: https://stackoverflow.com/a/11481887

Simply suggesting changes via issues might be the easiest way though.

**Reply by tsujan** [reply]
@Chiitoo's suggestion seems to me to be the best workaround for the current situation in GitHub, until GitHub fixes the issue.

**Reply by stefonarch** [reply]
As I tested if you clone the wiki by "fork" you end up here https://github.com/stefonarch/lxqt/wiki I didn't see any other "clone" possibility.

**Reply by Chiitoo** [reply]
Right, you can also clone it by doing something like this:

`git clone https://github.com/lxqt/lxqt.wiki.git`
